pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 120
-target-version = ['py37', 'py38', 'py39']
+target-version = ['py38', 'py39']
```

Comment:
i think we should add 3.10 as target version in here as well.

Reply:
I was also thinking about that. But let's keep it like this